docs/general/status.rst

```diff
@@ -40,6 +47,44 @@ consider waiting until it is more mature.
    Workstation that may hinder your ability to use it at any functional
    level.
 
+SecureDrop Workstation Feature Comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SecureDrop Workstation provides a faster and more streamlined way of interacting
+with sources and submissions, while preserving important security properties.
+That said, it is still under development, and not all features available on the
+Tails-based (existing) system are available in SecuredDrop Workstation. We are
+working towards feature parity, but in the meantime, there are some notable differences.
+
+The following features have not yet been implemented:
+
+* Viewing download progress/download speed
+* Cancelling downloads
+* Network speed indicators
+* Searching/filtering of sources by codename
+* User-selection of application in which to view submissions (submissions
+  are viewed in one default application depending on their filetype)
+* Printing of individual files submitted inside an archive
+* Scalability/performance improvements (support for large numbers of sources)
+
+In addition, there are some filetypes that are not yet supported for viewing
+on SecureDrop Workstation, although they can be exported for use on the Tails-
+based Secure Viewing Station.
+
+Currently-supported filetypes include:
+
+* .txt, .csv, .pdf
+* Microsoft Word files (.doc, .docx, .xls, .xlsx, .ppt, .pptx)
+* OpenDocument files (.odt, .ods, .odp)
+* Audio: .mp3, .mp4, .mpeg, .wav, .ogg (Ogg Vorbis)
+* Video: .mp4, .webm, .mov (Quicktime), .avi (Audio Video Interleave - Microsoft),
+  .wmv (Windows Media Video)
+* Image: .gif, .png, .jpeg, .tiff, .svg, .ico, .webm
+* Compressed archives (.zip, .tar.gz), although printer support for files inside
+  an archive is still to be implemented
+
+A full list of supported filetypes can be found `here <https://github.com/freedomofpress/securedrop-client/blob/4b7495cf8bd1dc11e2f52fb772fad55aaa01edab/workstation-config/mimeapps.list.sd-viewer>`_.
```

Comment:
Any specific reason to use a commit hash link instead of `main` or a release tag?

Reply:
Oh, that's a mistake, thanks - wanted just https://github.com/freedomofpress/securedrop-client/blob/main/workstation-config/mimeapps.list.sd-viewer